syscall_sysv.go

```diff
@@ -136,9 +133,34 @@ func callbackWrap(a *callbackArgs) {
 	fnType := fn.Type()
 	args := make([]reflect.Value, fnType.NumIn())
 	frame := (*[callbackMaxFrame]uintptr)(a.args)
+	var floatsN int
+	// offset the integer position by the number
+	// of floatsN because in the frame it starts with the float
+	// registers followed by the integer and then the stack after that.
```

Comment:
Is this the explanation about the variable intsN?

Reply:
yes